src/briefcase/commands/new.py

```diff
@@ -117,6 +106,33 @@ def validate_app_name(self, candidate):
 
         return True
 
+    def make_class_name(self, formal_name):
+        """
+        Construct a candidate class name from a formal name.
+
+        :param formal_name: The formal name
+        :returns: The candidate app name; will appear blank if only non-latin characters
+        were entered in the formal_name
+        """
+        formal_name.title()
```

Comment:
This doesn't do anything. You're creating a title case version the string, and then discarding it.

Reply:
I went ahead and include the title  method in the return statement. This is actually needed if the formal name is spelled with two lowercase words. I added a test to the test_make_class_name to show an example of this scenario where "hello world" formal name is changed to "HelloWorld" class name.